Algorithm.CSharp/HistoryWithSymbolChangesRegressionAlgorithm.cs

```diff
@@ -39,17 +39,18 @@ public override void Initialize()
             var history = History(new [] {symbol}, TimeSpan.FromDays(5700), Resolution.Daily).ToList();
             Debug($"{Time} - history.Count: {history.Count}");
 
-            const int expectedSliceCount = 3926;
+            const int expectedSliceCount = 3935;
```

Comment:
Because history is now allowed the aux data it changes our expected slice count. Bar count stays the same because aux data is not in `slice.Bars`

Reply:
History shouldn't be emitting auxiliary data points, behavior shouldn't change around this I believe

Reply:
Gotcha, placed a filter in the history providers filter